src/js_list.hpp

```diff
@@ -47,6 +47,7 @@ class List : public realm::List {
 
 template<typename T>
 struct ListClass : ClassDefinition<T, realm::js::List<T>, CollectionClass<T>> {
+    using Type = T;
```

Comment:
this seems unused

Reply:
It's used inside `compute_aggregate_on_collection`: `get_internal<typename T::Type, T>(this_object)`